app/services/output_manager/help_page.py

```diff
@@ -4,16 +4,30 @@
 
 import enum
 
+import pkg_resources
+
+import app.services.output_manager.message_handler as mhandler
 from app.resources.custom_help import HelpPage
+from app.utils.aggregated import get_latest_cli_version
 
 help_msg = HelpPage.page
-update = help_msg.get('update', 'default update message')
-new_release = ''
-for k, v in update.items():
-    if k != 'version':
-        new_release += f' {k}. {v}\n\n'
 
-update_message = f"\033[92mWhat's new (Version {update.get('version')}):\n\n" + new_release + '\033[0m'
+
+def get_cli_help_message():
+    update = help_msg.get('update', 'default update message')
+    new_release = ''
+    for k, v in update.items():
+        if k != 'version':
+            new_release += f' {k}. {v}\n\n'
+
+    update_message = f"\033[92mWhat's new (Version {update.get('version')}):\n\n" + new_release + '\033[0m'
+
+    # message user if there is a newer version of the CLI
+    latest_version = get_latest_cli_version()
+    if pkg_resources.get_distribution('app').version < latest_version:
```

Comment:
You can't rely on string comparisons for semantic versions. For example:

```python
versions = ['1.0.0', '2.0.0', '10.0.0']
versions.sort()
print(versions) # ['1.0.0', '10.0.0', '2.0.0']
```

Reply:
ahh that is right!! will update to `from packaging.version import Version`